velox/dwio/dwrf/reader/SelectiveStructColumnReader.cpp

```diff
@@ -33,7 +33,8 @@ SelectiveStructColumnReader::SelectiveStructColumnReader(
           scanSpec,
           dataType->type,
           std::move(flatMapContext)),
-      requestedType_{requestedType} {
+      requestedType_{requestedType},
+      debugString_(getExceptionContext().message()) {
```

Comment:
Why does SelectiveStructColumnReader get its debugString_ from the current ExceptionContext? Is this ExceptionContext supposed to be the one set in TableScan::getOutput()? Might there be cases where the current ExceptionContext is not set by TableScan as intended?